charts/zeebe-benchmark/templates/clients-service.yml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: clients
+  labels:
+    zeebe.io/component: client
+    {{- include "zeebe-benchmark.labels" . | nindent 4 }}
+spec:
+  selector:
+    zeebe.io/component: client
+  ports:
+  - name: http
+    port: 9600
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: clients
+  labels:
+    release: monitoring
+    {{- include "zeebe-benchmark.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      zeebe.io/component: client
+  endpoints:
+  - honorLabels: true
+    interval: 10s
```

Comment:
This interval is different to our other monitor or? Wasn't it before 30 seconds?

If we want to change it we should adjust it on grafana config, please we need to consider that it takes more space right?